app/views/optOut/ConfirmedOptOut.scala.html

```diff
@@ -88,6 +78,37 @@ <h2 class="govuk-heading-m">@messages("optout.confirmedOptOut.report-quarterly-n
     </ul>
 }
 
+
+@yourReportingFrequencyBlock = @{
+    val reportingFrequencyUrl = controllers.routes.ReportingFrequencyPageController.show().url
```

Comment:
might be easy to pass this URL as a parameter to corresponding viewModel to reduce dependency etc